### DISCUSSION TITLE
upgrade FUXA version and import database

### DISCUSSION THREAD

**Discussion by artofegg** [frangoteam/FUXA#578] [Q&A]
how to use new code from git ,try do  `npm install -g --unsafe-perm @frangoteam/fuxa`,but the version is 1.1.9. is not the newest code to fix history show problem. start up fuxa use parameters can assign database files , please show the method, thx!

**Comment by malcore10** [comment]
[Installing-and-Running](https://github.com/frangoteam/FUXA/wiki/Installing-and-Running) the second option

**Comment by artofegg** [comment]
hi malcore10 , thanks for your reply , how to use the existing database file in the first way and the second way? Conflict?

**Reply by malcore10** [reply]
you replace the server folders:
 - _appdata
 - _db

**Reply by artofegg** [reply]
thank you for your new message~
